README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Yes, ```In the real world, if using real robot, if it's possible that the client machine running rviz is far away from the real robot?```
That is a posible scenario.  For this PR, lets first try to work on the 4 scenarios you described above and then we'll work on the scenario where rviz runs on a client machine separate from the client that runs the robot and sensor.